Sources/RxMoya/Single+Response.swift

```diff
@@ -7,13 +7,20 @@ import RxSwift
 /// Extension for processing raw NSData generated by network access.
 extension PrimitiveSequence where TraitType == SingleTrait, ElementType == Response {
 
-    /// Filters out responses that don't fall within the given range, generating errors when others are encountered.
+    /// Filters out responses that don't fall within the given closed range, generating errors when others are encountered.
     public func filter(statusCodes: ClosedRange<Int>) -> Single<ElementType> {
         return flatMap { response -> Single<ElementType> in
             return Single.just(try response.filter(statusCodes: statusCodes))
         }
     }
 
+    /// Filters out responses that don't fall within the given range, generating errors when others are encountered.
+    public func filter(statusCodes: Range<Int>) -> Single<ElementType> {
+        return flatMap { response -> Single<ElementType> in
```

Comment:
Can we not forward this to the `ClosedRange` `filter` to prevent duplicating the implementation?

Reply:
Sure :))